src/Eto.Mac/Eto.Mac64.csproj

```diff
@@ -17,12 +17,14 @@
     <PackageDescription>
 This is the 64-bit MonoMac platform for Eto.Forms UI Framework.
       
-Eto.Platform.Mac64 uses a modified version of the open source MonoMac for 64-bit, which allows you to create macOS application bundles from any platform.  However, it does require mono to be installed when running on macOS.
+Eto.Platform.Mac64 uses a modified version of the open source MonoMac for 64-bit, which allows you to create macOS application bundles from any platform.
       
-Use Eto.Platform.XamMac2 if you want to bundle mono inside your .app, however it requires Visual Studio on a Mac.
-      
-You can create your own .app bundle to run your app on OS X, without an OS X machine. This is included as the MyApp.app folder. Read MyApp.app\Contents\MonoBundle\README.txt for instructions on next steps.
+This package creates an .app bundle to run your app on macOS, even on Windows or Linux.
       
+By default, the mono or .NET Core runtime will be bundled in Release configuration so your app won't require extra dependencies. To bundle mono, it must be installed on the development machine.
+
+Use Eto.Platform.XamMac2 if you want to distribute your app to the Mac App Store, though it requires VS for Mac.
```

Comment:
Hello @cwensley , and thank you for this project!
A question to this comment, why can not .NET Core-bundled app be distributed on app store?

Reply:
Hey @rafntor, thanks for the inquiry!

The App Store has many requirements that the .NET Core builds will most likely not pass, although admittedly I haven't tried.  You'd also have to do many manual steps to package it up, submit it for notarization, etc. which could be a painful process and isn't documented.

I would welcome anyone that is inclined to try to go through those steps, but if you simply want your app on the App Store, using Xamarin.Mac guarantees it will work and will be a better/easier experience, it's documented, and supported.  

Hope this helps!
Curtis.